services/src/unit-test/java/uk/gov/moj/sdt/services/RetryMessageAlreadySentServiceTest.java

```diff
@@ -232,158 +235,139 @@ public void queuePendingMessageExceededForwardingAttempts() {
      * flag is ignored and not picked up by the service method.
      */
     @Test
-    public void queuePendingDLQMessageExceededForwardingAttempts() {
-        final String sdtRequestRef = "TEST_1";
-        final String dlqSdtRequestRef = "TEST_2";
+    void queuePendingDLQMessageExceededForwardingAttempts() {
+        final String sdtRequestRef = TEST_1;
+        final String dlqSdtRequestRef = TEST_2;

```

Comment:
As a constants have been created for the SDT request values these variables are redundant. They can be removed and the constants used in their place.

Reply:
removed